crates/nu-command/src/network/fetch.rs

```diff
@@ -63,6 +64,22 @@ impl Command for SubCommand {
                 "fetch contents as text rather than a table",
                 Some('r'),
             )
+            .named(
+                "output",
+                SyntaxShape::Filepath,
+                "save contents into a file",
+                Some('o'),
```

Comment:
Bikeshed: 
`-o` to me is either for specifing an output file or `--output` signals to me that it switches regular stdout output.

Whatabout `-f`/`--file` instead

Reply:
that works for me as well.

Reply:
curl use `--output`, wget use `--output-file`, aria2c use `--out`, I think `--file` is not a good choice here, `--output-file` should be better if we want to have `file` in the flag

Reply:
Yeah that consistency is probably more valuable. I checked `$nu.scope.commands.signature | flatten | where short_flag == f` and `$nu.scope.commands.signature | flatten | where short_flag == o` and it seems neither of them are used in this context so we probably should follow general precedence.